expressjs/src/routes/events/printer.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @typedef {import('cloudevents').CloudEvent} CloudEvent
+ */
+
+class PrettyPrinter {
+
+  /**
+   * Prints a CloudEvent in human-readable format.
+   *
+   * @param {CloudEvent} ce - the CloudEvent to receive
+   * @returns {string} - the human-readable representation of the CloudEvent
+   */
+  print(ce) {
+    let valid = 'valid'
+    try {
+      ce.validate()
+    } catch (err) {
+      valid = 'invalid'
+    }
+    let buf = '☁️  cloudevents.Event\n'
+    buf += `Validation: ${valid}\n`
+    const attr = attributes(ce)
+    buf += printAttr(attr)
+    buf += printExts(ce, attr)
+    buf += printData(ce)
+    return buf
+  }
+
+}
+
+/**
+ * @param {CloudEvent} ce - the CloudEvent
+ * @returns {Object} - the CloudEvent attributes
+ * @private
+ */
+function attributes(ce) {
+  const attr = {
+    specversion: ce.specversion,
+    type: ce.type,
+    source: ce.source,
+    id: ce.id,
+  }
+  if (ce.subject) {
+    attr.subject = ce.subject
+  }
+  if (ce.time) {
+    attr.time = ce.time
+  }
+  if (ce.datacontentencoding) {
+    attr.datacontentencoding = ce.datacontentencoding
+  }
+  if (ce.dataschema) {
+    attr.dataschema = ce.dataschema
+  }
+  if (ce.datacontenttype) {
+    attr.datacontenttype = ce.datacontenttype
+  }
+  return attr
+}
+
+/**
+ * @param {Object} attr - the CloudEvent attributes
+ * @returns {string} - the buffer
+ */
+function printAttr(attr) {
+  let buf = 'Context Attributes,\n'
```

Comment:
Should there be `:` after Context Attributes? 
Running the tests, it gives me this:
```
Received event:
☁️  cloudevents.Event
Validation: valid
Context Attributes,
  specversion: 1.0
  type: com.redhat.openshift.knative.showcase.Hello
  source: //localhost/dev
  id: 780ed118-9509-4d7b-a612-97f278eb2b55
  time: 2022-02-24T02:15:27.522Z
  datacontenttype: application/json
Data,
  {
    "greeting": "Welcome",
    "who": "Developer31",
    "number": 64
  }

```

Reply:
@mgencur fwiw, this is the format that the golang cloudevents SDK prints events. I'm not saying one way or the other how it should be printed here other than to note that there is precedent for this format.

Reply:
I guess I should have read all of the comments, including @cardil's first. 🤣

Reply:
For the original viewer app, the output was just to print the event using the `String()` method. https://github.com/cloudevents/sdk-go/blob/ff0a142752d3253c3b974246c07f450421714f23/v2/event/event.go#L70-L97

Reply:
Thanks, ok good to know. I guess this is alright. Not saying I like it but we can use it as it's used elsewhere.